ideas/100-smooth-chat-ux.md

```diff
@@ -0,0 +1,119 @@
+<!-- A swarm requires at minimum 3 contributors and 1 evaluator/tester -->
+<!-- 'Contributor' should be replaced with a descriptive role type. -->
+- Lead Contributor: @ricardosaavedra
+- Testing & Evaluation: <!-- @username -->
+- Contributor: <!-- @username -->
+- Contributor: <!-- @username -->
+- PM: <!--- @username -->
+- UX (if relevant): <!-- @username -->
+<!-- - Contributor: @username -->
+
+## Workflow and day-to-day activities
+<!-- A short (~200 word) description and motivation of the Idea. Without clear explanation the Idea should not proceed. Can include User Stories -->
+<!-- Testing/Evaluation role accepts responsbility to checkin at Goal dates, -->
+<!-- forces discussion to continue implementation or recommend disband and post-mortem. -->
+Every beginning of a new cycle (you can call it sprint if you'd like), we will decide together what to focus on for the upcoming two weeks as well as discussing current problems, technical challenges and the best solutions (trade-off between implementation cost and UX).
+As ideas and design solutions start to emerge, we will get together to validate/critique the proposed solutions until we arrive at a group consensus.
+
+
+### Product Description
+<!-- What functionality are you adding? What will this look like from a user perspective? Why is this important? -->
+
+**V1 - quick fixes**
+- Chat view / message status
+- Chat view / last seen 
+- Chat view / user page 
+- Chat view / minior UI fixes 
+- Chat view / message space optimization
+- Chat view / timestamps
+- Chat view / adding/blocking new contact flow
+- Keyboard / use cases behaviors
+- Keyboard / maximum message width
+- Keyboard / fixing console position
+- Offline states and error handling 
+
+**V2 - group chat improvements**
+
+- Group chat view / chat metadata
+- Group chat view / message space optimization 
+- Group chat / timestamps
+- Group chat / seen by logic and documentation
+- Group chat / grouping messages 
+- Group chat view / top bar behavior
+
+**V3 - more engaging and visually clear home screen**
+
+- Home screen / starting a new chat flow
+- Home screen / visual consistency fixes
+- Home screen / search function 
+- Home screen / loading messages experience
+
+For more details about each one of those fixes, please visit: https://docs.google.com/document/d/1AyU9mWMG1MxNPPBr1kidIl7eMJzPhbUjJjUNBBxZpg8/edit
+
+### Team & Dependencies
+<!-- Are there bugs or feature requests in other repositories that are part of this Idea? -->
+<!-- There is no approval unless the idea requires to be reviewed by supporting organelles (Financial, Hiring, or Design). -->
+<!-- The Swarm must develop a fully fleshed out Requirements document for the idea to proceed, to the satisfaction of participants. -->
+Ideally, we would have at least 2 developers (Whisper, Clojure) in the team acting as a stakeholder in the process.
```

Comment:
What do you mean by Whisper developer? What knowledge is required? I believe Clojure devs hopefully have enough knowledge on application protocol layer for this.

What does the overlap with this and 99 look like btw?